stake-o-matic.sh

```diff
@@ -52,18 +48,32 @@ MAINNET_BETA_ARGS=(
   $MARKDOWN
   $CONFIRM
   $REQUIRE_CLASSIFICATION
+)
+
+# shellcheck disable=SC2206
+NOT_A_STAKE_POOL_ARGS=(
   stake-pool-v0
   --baseline-stake-amount ${BASELINE_STAKE_AMOUNT:?}
   ${RESERVE_ACCOUNT_ADDRESS:?}
   ${STAKE_AUTHORITY_KEYPAIR:?}
 )
 
-if [[ $CLUSTER == "testnet" ]]; then
-  ./solana-stake-o-matic "${TESTNET_ARGS[@]}"
+# shellcheck disable=SC2206
+STAKE_POOL_ARGS=(
+  stake-pool
+  --baseline-stake-amount ${BASELINE_STAKE_AMOUNT:?}
+  ${RESERVE_ACCOUNT_ADDRESS:?}
```

Comment:
This variable is poorly named and should really be `${STAKE_POOL_ADDRESS:?}`, but it seems like that wouldn't place with the dynamic setup

Reply:
You can make the renaming, and just add a fourth cluster config `mainnet-beta-stake-pool` that uses it, and `mainnet-beta` can continue to use `stake-pool-v0` args